GoogleForCreators/web-stories-wp
# Update useRovingTabIndex to focus outer element instead of firstChild

## Summary

Update useRovingTabIndex to focus outer element instead of firstChild.

This will allow useRovingTabIndex to be re-used for the category pills in the media3p categories list.

## Testing Instructions

* Open media gallery from main.
* Open media gallery from this PR.
* Use the arrows to navigate around locally uploaded media, and verify both behave / appear the same.
* Use the arrows to navigate around locally uploaded videos, and verify both behave / appear the same.
* Use the arrows to navigate around Unsplash images, and verify both behave / appear the same.

---



Addresses #3398
